public/app/plugins/datasource/prometheus/configuration/PromSettings.tsx

```diff
@@ -335,6 +337,50 @@ export const PromSettings = (props: Props) => {
             />
           </div>
         </div>
+
+        <div className="gf-form-inline">
+          <div className="gf-form max-width-30">
+            <FormField
+              label="Incremental querying (beta)"
```

Comment:
Just asking, should we use [this](https://developers.grafana.com/ui/latest/index.html?path=/story/data-display-badge--basic) to show beta notice instead of having `(beta)`?

Reply:
<img width="499" alt="image" src="https://user-images.githubusercontent.com/109082771/229118854-c6e2a4bd-ef5e-4418-8ac6-2d91fe285bcb.png">
I would love to! But where would we put it? We can't add JSX to the input labels without refactoring a bunch of stuff. I added it after the field, but I dunno if that looks any better. @catherineymgui?